<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Route</class>
 <widget class="QWidget" name="Route">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>564</width>
    <height>100</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>525</width>
    <height>100</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>568</width>
    <height>100</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="3">
    <widget class="Line" name="line_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="4">
    <widget class="QLabel" name="labe_dateTime">
     <property name="text">
      <string>Дата и время</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="Line" name="line_5">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="4">
    <widget class="Line" name="line_7">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="Line" name="line_6">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label_city_from">
     <property name="text">
      <string>Пункт отправления</string>
     </property>
    </widget>
   </item>
   <item row="1" column="6">
    <widget class="QPushButton" name="pushButton_buy">
     <property name="text">
      <string>Купить</string>
     </property>
    </widget>
   </item>
   <item row="0" column="6">
    <widget class="QLabel" name="labe_cnt_seats">
     <property name="text">
      <string>Количество мест</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>--&gt;</string>
     </property>
    </widget>
   </item>
   <item row="0" column="5">
    <widget class="Line" name="line_4">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QLabel" name="label_city_to">
     <property name="text">
      <string>Пункт назначения</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
